smac/optimizer/ei_optimization.py

```diff
@@ -281,7 +283,9 @@ def _one_iter(
             # Get one exchange neighborhood returns an iterator (in contrast of
             # the previously returned list).
             all_neighbors = get_one_exchange_neighbourhood(
-                incumbent, seed=self.rng.seed())
+                incumbent, seed=self.rng.randint(MAXINT))
+
+            logging.warning('__test_rng__' + str(self.rng.randint(MAXINT)))
```

Comment:
It that output here on purpose? If yes, could you make it clear what the warning is for?

Reply:
I guess this comment got lost :-)